pandas/core/groupby/groupby.py

```diff
@@ -3534,6 +3559,16 @@ def resample(self, rule, *args, **kwargs) -> Resampler:
         ----------
         rule : str or DateOffset
             The offset string or object representing target grouper conversion.
+        include_groups : bool, default True
+            When True, will attempt to include the groupings in the operation in
+            the case that they are columns of the DataFrame. If this raises a
+            TypeError, the result will be computed with the groupings excluded.
```

Comment:
Can you also describe here what happens when `include_groups=False`

Reply:
Done.